example/template.env

```diff
@@ -47,5 +47,7 @@ QUICKSTATEMENTS_HOST=quickstatements.svc
 QUICKSTATEMENTS_PORT=8840
 
 ## ElasticSearch
+## Comment out MW_ELASTIC_HOST to disable ElasticsSearch
+## See https://github.com/wmde/wikibase-release-pipeline/blob/main/Docker/build/WikibaseBundle/LocalSettings.d.template/WikibaseCirrusSearch.php#L6
```

Comment:
That's cool. Alternatively one could also consider linking to the file and the line in a specific state of the repository (commit), so that the link certainly always point to what is meant here